### ISSUE TITLE
Word report: justify text orientation

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite.ReportingEngine#747]
![image](https://user-images.githubusercontent.com/25903040/161032197-b507baa6-2c8c-4314-81e3-228b074d6e96.png)

vs

![image](https://user-images.githubusercontent.com/25903040/161032073-1f2e698d-0e7e-4551-8dc9-3c4e744065a0.png)

**Comment by Yuri05** [comment]
I don't see a difference...

**Comment by PavelBal** [comment]
![image](https://user-images.githubusercontent.com/25903040/161042197-1937867c-5c95-4574-8b74-ef6d49f254e2.png)


vs

![image](https://user-images.githubusercontent.com/25903040/161042269-c9d106d2-f840-4e40-8374-54e284e3ef3d.png)


"Justify" looks calmer.

**Comment by Yuri05** [comment]
Still don't understand the issue. Are you complaining about the _text font_?

**Comment by PavelBal** [comment]
No, text alignment:

![image](https://user-images.githubusercontent.com/25903040/161043559-3b621f64-a8ba-4f55-9961-72bddb9931cd.png)


it is "Left" by default, I propose "Justify"

**Comment by Yuri05** [comment]
Ok, got it know. Never knew that option is called "Justify" :)
Then... the same as for all other Word formatting applies: https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/issues/748#issuecomment-1084426873

**Comment by pchelle** [comment]
Setting default alignment to Justify can be done by **Modifying** the styles of the reference document.

After some investigation, it is also possible to create new styles in the docx reference document with specific alignments (left, center, right, justify).
These custom styles can then be included in the markdown document as suggested by the [Pandoc documentation](https://pandoc.org/MANUAL.html#output) and applied to the word report by Pandoc.

For instance, if you create a style named "Centered" in your reference document that use a center alignment, the line below in the markdown document will create a centered paragraph.

```md
::: {custom-style="Centered"}
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
:::
```

**Comment by pchelle** [comment]
Fixed by PR [#1065](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/pull/1065)

There is an article documenting how to [leverage html tags for word conversion](https://www.open-systems-pharmacology.org/OSPSuite.ReportingEngine/dev/articles/word-report.html#tips-for-report-conversion)
